catalyst-gateway/Earthfile

```diff
@@ -1,112 +1,51 @@
-VERSION 0.7
+VERSION --try --global-cache 0.7
 
 #cspell: words rustfmt toolsets USERARCH
 
 # Set up our target toolchains, and copy our files.
 builder:
-    FROM github.com/input-output-hk/catalyst-ci/earthly/rust:v2.0.14+rust-base
-
-    DO github.com/input-output-hk/catalyst-ci/earthly/rust:v2.0.14+SETUP --toolchain=rust-toolchain.toml
+    DO github.com/input-output-hk/catalyst-ci/earthly/rust:v2.6.0+SETUP --toolchain=rust-toolchain.toml
```

Comment:
Actually you can remove `--toolchain=rust-toolchain.toml` argument specification, because it has a default value, the same one which you are passing. But don't have a strong position here

Reply:
We don't need the `targets = ["x86_64-unknown-linux-musl"]`? @Mr-Leshiy

Reply:
we need them, but my point here that the `+SETUP` function has a default value of the `toolchain` argument which precisely equals to the `rust-toolchain.toml`, so it means that for this particular case you don't need to specify this argument and correct `rust-toolchain.toml` file still will picked up

Look at https://github.com/input-output-hk/catalyst-ci/blob/45ff4067986c24b12f8968b7964efd2fafcbbf61/examples/rust/Earthfile#L7

https://github.com/input-output-hk/catalyst-ci/blob/45ff4067986c24b12f8968b7964efd2fafcbbf61/earthly/rust/Earthfile#L109